packages/python-packages/api-stub-generator/apistub/nodes/_class_node.py

```diff
@@ -107,6 +106,24 @@ def _should_include_function(self, func_obj):
 
         return False
 
+    def _handle_class_variable(self, child_obj, name, *, type_string=None, value=None):
+        # Add any public class level variables
+        _allowed_types = [str, int, dict, list, float, bool]
+        if not any(isinstance(child_obj, x) for x in _allowed_types):
```

Comment:
`isinstance` takes a tuple of types to check. See https://docs.python.org/3/library/functions.html#isinstance. No need to do a generator-through-a-list...

Reply:
Updated